ScnSoft-Education/LeetCodeTraining.Tests/RemoveDuplicatesTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using LeetCodeTraining.RemoveDuplicates;
+
+namespace LeetCodeTraining.Tests
+{
+    public class RemoveDuplicatesTests
+    {
+        [Theory]
+        [InlineData(new int[] { 1, 1, 2 }, 2)]
+        [InlineData(new int[] { 0, 0, 1, 1, 1, 2, 2, 3, 3, 4 }, 5)]
+        [InlineData(new int[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 }, 10)]
+        [InlineData(new int[] { 1 }, 1)]
+        [InlineData(new int[] { 1, 1, 1, 1, 1, 1 }, 1)]
+        public void RemoveDuplicatesTest(int[] nums, int expectedResult)
+        {
+            //Act
+            int result = RemoveDuplicatesSolution.RemoveDuplicates(nums);
```

Comment:
Sorry, forgot  to mention.

Can you please implement also checking that first k values of result nums are unique values as expected in accordance with task requirements?

> Consider the number of unique elements of nums to be k, to get accepted, you need to do the following things:
> 
> Change the array nums such that the first k elements of nums contain the unique elements in the order they were present in nums initially. The remaining elements of nums are not important as well as the size of nums.
> Return k.